.appveyor.yml

```diff
@@ -0,0 +1,20 @@
+environment:
+  matrix:
+  - nodejs_version: 8
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - npm install -g npm # Force using the latest npm to get dedupe during install
```

Comment:
Why? I'd assume we want to use the version of npm that comes with the node release.

Reply:
as the comment states - `Force using the latest npm to get dedupe during install`. Not sure if it's actual, I'm OK to remove.

Reply:
Lets start with out it and add it back if needed